source/views/dictionary/report/editor.js

```diff
@@ -65,13 +65,11 @@ export class DictionaryEditorView extends React.PureComponent<Props, State> {
 				keyboardDismissMode="on-drag"
 				keyboardShouldPersistTaps="always"
 			>
-				<View style={styles.helpWrapper}>
-					<Text style={styles.helpTitle}>Thanks for spotting a problem!</Text>
-					<Text style={styles.helpDescription}>
-						If you could tell us what the word and definition should be,
-						we&rsquo;d greatly appreciate it.
-					</Text>
-				</View>
+				<InfoHeader
+					message="If you could tell us what the word and definition should be,
+                        we&rsquo;d greatly appreciate it."
```

Comment:
Same question about newlines

Reply:
I have changed this under the assumption that this was not intentional.